src/gui/calibration.cpp

```diff
@@ -64,8 +64,8 @@ Calibration::Calibration(InputDevice *joystick, QWidget *parent)
 
     QPointer<JoyControlStick> controlstick = currentJoystick->getActiveSetJoystick()->getJoyStick(0);
     this->stick = controlstick.data();
-    calibrated = this->stick->wasCalibrated();
-    ui->Information->setText(stick->getCalibrationSummary());
+    // calibrated = this->stick->wasCalibrated();
```

Comment:
I have to look at this again. It seems like I commented out some code without implementing the replacement.

Reply:
I removed the code instead of commenting it out in the updated changes.
This is OK here because "calibration.cpp" will be implemented in a following commit.